test/integration/patientSay.js

```diff
@@ -0,0 +1,36 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const cheerio = require('cheerio');
+const app = require('../../server');
+const constants = require('../../app/lib/constants');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+describe('online services', () => {
+  const ratingsReviewsClass = '.gp-ratings-reviews';
+  const patientSurveyClass = '.gp-patient-survey';
+
+  describe('when links are avaiable', () => {
```

Comment:
I don't think this describe block is required with the rewording of the top level `describe` block and the subsequent `it` block.